core/src/main/java/software/amazon/awssdk/core/http/DefaultErrorResponseHandler.java

```diff
@@ -60,25 +60,25 @@
      *                         response.
      */
     public DefaultErrorResponseHandler(
-            List<Unmarshaller<AmazonServiceException, Node>> unmarshallerList) {
+            List<Unmarshaller<SdkServiceException, Node>> unmarshallerList) {
         this.unmarshallerList = unmarshallerList;
     }
 
     @Override
-    public AmazonServiceException handle(HttpResponse errorResponse,
-                                         ExecutionAttributes executionAttributes) throws Exception {
-        AmazonServiceException ase = createAse(errorResponse);
-        if (ase == null) {
+    public SdkServiceException handle(HttpResponse errorResponse,
+                                      ExecutionAttributes executionAttributes) throws Exception {
+        SdkServiceException exception = createAse(errorResponse);
+        if (exception == null) {
             throw new SdkClientException("Unable to unmarshall error response from service");
         }
-        ase.setHttpHeaders(errorResponse.getHeaders());
-        if (StringUtils.isNullOrEmpty(ase.getErrorCode())) {
-            ase.setErrorCode(errorResponse.getStatusCode() + " " + errorResponse.getStatusText());
+        exception.headers(errorResponse.getHeaders());
+        if (StringUtils.isNullOrEmpty(exception.errorCode())) {
+            exception.errorCode(errorResponse.getStatusCode() + " " + errorResponse.getStatusText());
         }
-        return ase;
+        return exception;
     }
 
-    private AmazonServiceException createAse(HttpResponse errorResponse) throws Exception {
+    private SdkServiceException createAse(HttpResponse errorResponse) throws Exception {
```

Comment:
might want to rename this method since it's not returning `AmazonServiceException` anymore. `createServiceException`?

Reply:
Or createSse, that won't be ambiguous at all ;)

Reply:
Ah, yep, will update. Thought I had most of them but found a new set. Hard to find all the cases of "ase" and all of it's casing throughout the code.